src/test/mir-opt/dataflow-const-prop/cell.main.DataflowConstProp.diff

```diff
@@ -0,0 +1,12 @@
+- // MIR for `main` before DataflowConstProp
++ // MIR for `main` after DataflowConstProp
+  
+  fn main() -> () {
+      let mut _0: ();                      // return place in scope 0 at $DIR/cell.rs:+0:11: +0:11
+  
+      bb0: {
+          _0 = const ();                   // scope 0 at $DIR/cell.rs:+0:11: +2:2
+          return;                          // scope 0 at $DIR/cell.rs:+2:2: +2:2
+      }
+  }
+  
```

Comment:
This test doesn't test anything

Reply:
Yes, thanks. It mainly was a reminder for me to think about `Cell`/`UnsafeCell`. But I have added this to the todo list of the PR, so this "test" can be removed for now.